react/__tests__/Carousel.test.js

```diff
@@ -5,63 +5,65 @@ import { render } from 'react-testing-library'
 import Carousel from '../Carousel'
 
 describe('Carousel component', () => {
-  let wrapper
-
-  const props = {
-    height: 440,
-    mobileHeight: 339,
-    showArrows: true,
-    showDots: true,
-    autoplay: true,
-    autoplaySpeed: 4,
-    numberOfBanners: 3,
-    runtime: {
-      hints: {}
-    },
-    banners: [
-      {
-        image:
-          'https://raw.githubusercontent.com/vtex-apps/carousel/master/images/banners-01.png',
-        externalRoute: true,
-        description: 'banner',
-        url:
-          'https://raw.githubusercontent.com/vtex-apps/carousel/master/images/banners-01.png',
-      },
-      {
-        image:
-          'https://raw.githubusercontent.com/vtex-apps/carousel/master/images/banners-02.png',
-        externalRoute: true,
-        description: 'david',
-        url:
-          'https://raw.githubusercontent.com/vtex-apps/carousel/master/images/banners-01.png',
+  const renderComponent = () => {
+    const props = {
+      height: 440,
+      mobileHeight: 339,
+      showArrows: true,
+      showDots: true,
+      autoplay: true,
+      autoplaySpeed: 4,
+      numberOfBanners: 3,
+      runtime: {
```

Comment:
You can change your `withRuntimeContext` mock to add runtime instead of adding here.
Example: https://github.com/vtex-apps/product-details/blob/master/react/__mocks__/vtex.render-runtime.js#L7

Reply:
As a matter of fact, i mocked it on my runtime mock and forgot to remove from here  😄